doc/htmldoc/installation/cmake_options.rst

```diff
@@ -159,19 +161,30 @@ neurons, writing to ASCII files might become prohibitively slow due to
 the large number of resulting files. By installing the `SIONlib
 library <http://www.fz-juelich.de/jsc/sionlib>`_ and supplying its
 installation path to the ``-Dwith-sionlib=<path>`` option when calling
-`cmake`, you can enable the :ref:`recording backend for binary files
+``cmake``, you can enable the :ref:`recording backend for binary files
 <recording_backends>`, which solves this problem.
 
-If you compiled NEST with support for MPI and also want to run the
-corresponding tests, you have to tell it about how your
-``mpirun``/``mpiexec`` command works by defining the ``mpirun``
-function in your ``~/.nestrc`` file. The file already contains an
-example implementation that should work with the `OpenMPI
-<http://www.openmpi.org>`__ implementation. For more details, see the
-documentation on the :ref:`config_options`.
+In order to run the distributed tests upon ``make installcheck``, NEST
+needs to know how to execute the launcher of your MPI implementation.
+CMake is usually able to detect the command line for this, but you can
+customize it using the follwing configuration variables (common
+defaults are shown below)::
 
-See the :ref:`parallel_computing` to learn how to execute
-threaded and distributed simulations with NEST.
+    -DMPIEXEC=/usr/bin/mpiexec
+    -DMPIEXEC_NUMPROCS_FLAG=-np
+    -DMPIEXEC_PREFLAGS=
```

Comment:
Should we mention `--oversubscribe` as a possible flag, maybe the most useful one for developers?

Reply:
I think we shouldn't. `--oversubscribe` is an implementation-detail of OpenMPI I'd rather keep out of NEST's documentation.  The testsuite now adds it automatically when OpenMPI is used and SLI's `mpirun` command is not used anywhere else.

Reply:
OK